### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard:Focus is not logical after cancel button

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9227]
[“Check out Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.4.3;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.4.3

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Server Group' icon.
4. Navigate to Cancel button.

**Actual:**
On navigating to the cancel button hit enter the next tab key focus is seen on the server icon in the left panel.

**Same issue is observed in the following page:**
1.Add new controller

**Expected:**
The focus should be on the button "new server group" button that opened the dialog.

**User Impact:**
Keyboard users will not receive a logical navigation sequence as well as the button that revealed the content will not be visible on the first tab.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard9227.zip](https://github.com/microsoft/azuredatastudio/files/4322648/A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard9227.zip)

**Comment by alanrenmsft** [comment]
this is our dialog, need to go back to previously focused element when dialog is dismissed

**Comment by alanrenmsft** [comment]
@abist , please refer to this PR: https://github.com/microsoft/vscode/pull/92969/files and make similar changes in our dialog to fix the issue.

**Comment by alanrenmsft** [comment]
I mean in the modal dialog base, so that all dialogs can get it.

**Comment by alanrenmsft** [comment]
I will look into it

**Comment by sspatil123** [comment]
GithubTags:#Closed; Version 1.41.1
![image](https://user-images.githubusercontent.com/85863999/216964877-95f6cdde-ee6c-4029-9686-71647b6a64cd.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
